programs/svm-spoke/src/instructions/bundle.rs

```diff
@@ -101,10 +108,13 @@ impl RelayerRefundLeaf {
 
 pub fn execute_relayer_refund_leaf<'info>(
     ctx: Context<'_, '_, '_, 'info, ExecuteRelayerRefundLeaf<'info>>,
-    root_bundle_id: u32,
-    relayer_refund_leaf: RelayerRefundLeaf,
-    proof: Vec<[u8; 32]>,
 ) -> Result<()> {
```

Comment:
Do you think it makes sense to have 2 versions of this function? one using the pre-loaded instructions and a classic (backup) version?

Reply:
I have not found yet a clean way to implement 2 external methods that use the same internal implementation in solana. This would require maintaining duplicate versions of most of the code.